--- sqlrelay-0.37.1/src/api/perl/Connection/Makefile~	2005-07-23 08:22:30.000000000 +0300
+++ sqlrelay-0.37.1/src/api/perl/Connection/Makefile	2006-09-12 11:23:38.636829444 +0300
@@ -9,7 +9,7 @@
 ifeq ($(OVERRIDEPERLSITEARCH),)
 PERLSITEARCH = $(DESTDIR)$(shell eval "`$(PERL) -V:sitearch`"; export sitearch; echo $$sitearch | sed -e "s|UNKNOWN||g" )
 else
-PERLSITEARCH = $(DESTDIR)$(PERLSITEARCH)
+PERLSITEARCH = $(DESTDIR)$(OVERRIDEPERLSITEARCH)
 endif
 PERL_INC = $(DESTDIR)$(shell eval "`$(PERL) -V:archlibexp`"; export archlibexp; echo "-I$$archlibexp/CORE" | sed -e "s|UNKNOWN||g" )
 PERL_LIB = $(DESTDIR)$(shell eval "`$(PERL) -V:privlibexp`"; export privlibexp; echo $$privlibexp | sed -e "s|UNKNOWN||g" )
--- sqlrelay-0.37.1/src/api/perl/Cursor/Makefile~	2005-07-23 08:22:30.000000000 +0300
+++ sqlrelay-0.37.1/src/api/perl/Cursor/Makefile	2006-09-12 11:24:21.577792882 +0300
@@ -9,7 +9,7 @@
 ifeq ($(OVERRIDEPERLSITEARCH),)
 PERLSITEARCH = $(DESTDIR)$(shell eval "`$(PERL) -V:sitearch`"; export sitearch; echo $$sitearch | sed -e "s|UNKNOWN||g" )
 else
-PERLSITEARCH = $(DESTDIR)$(PERLSITEARCH)
+PERLSITEARCH = $(DESTDIR)$(OVERRIDEPERLSITEARCH)
 endif
 PERL_INC = $(DESTDIR)$(shell eval "`$(PERL) -V:archlibexp`"; export archlibexp; echo "-I$$archlibexp/CORE" | sed -e "s|UNKNOWN||g" )
 PERL_LIB = $(DESTDIR)$(shell eval "`$(PERL) -V:privlibexp`"; export privlibexp; echo $$privlibexp | sed -e "s|UNKNOWN||g" )
